modules/tracing/mule-tracer-exporter-impl/src/main/java/org/mule/runtime/tracer/exporter/impl/OpenTelemetryTraceIdUtils.java

```diff
@@ -165,7 +170,8 @@ private static boolean parentSpanContextIsValid(InternalSpan parentInternalSpan)
     return false;
   }
 
-  public static Map<String, String> getContext(OpenTelemetrySpanExporter openTelemetrySpanExporter) {
+  public static Map<String, String> getContext(OpenTelemetrySpanExporter openTelemetrySpanExporter,
```

Comment:
I'm suggesting a naming change. Also, can we add JavaDoc? 
```suggestion
  public static Map<String, String> getDistributedTraceContext(OpenTelemetrySpanExporter openTelemetrySpanExporter,
```